cosmos/cosmwasm-liquidity/src/mapping.ts

```diff
@@ -0,0 +1,55 @@
+import { BigInt, cosmos, log } from "@graphprotocol/graph-ts";
+import { cosmwasm } from "@graphprotocol/juno-ts";
+import { Contract } from "../generated/schema";
+import { JSON } from "assemblyscript-json";
```

Comment:
I'm wondering if we could use this: https://thegraph.com/docs/en/developer/assemblyscript-api/#json-api=